app/timetables/models.py

```diff
@@ -39,3 +39,21 @@ def clean(self):
 
     def __str__(self):
         return self.name
+
+
+class Dish(ForceCapitalizeMixin, TimestampMixin):
+    """
+    Model representing Dish.
+
+    A dish represents the actual food served as a given 
+    course under an option of a meal on a cycle day in a timetable. 
+    E.g, Coconut rice garnished with fish stew and chicken or just 
+    Ice-cream.
+    """
+
+    name = models.CharField(max_length=60, unique=True)
```

Comment:
@andela-joyebanji: Though I do not have a case study for this right now, I think we might have an entry of a dish with name of more that 60 characters; 

> Coconut rice garnished with fish stew and chicken is 49 char long.